### PR TITLE
Add University of Idaho

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Egad. This is my first time.

File: _data/education.yml [hunk 1]
```diff
@@ -284,6 +284,16 @@ websites:
...
 
+    - name: University of Idaho
+      url: https://www.uidaho.edu
+      img: university-of-idaho_32x32.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      hardware: Yes
+      software: Yes
+      doc: https://support.uidaho.edu/TDClient/Requests/ServiceDet?ID=658
```
Comment (mxxcon) [inline _data/education.yml]: Perhaps this is a better link for docs? https://support.uidaho.edu/TDClient/KB/ArticleDet?ID=109

Comment (ghost) [reply _data/education.yml]: That also works.

**Review by stephengroat — DISMISSED** [review]

**Comment by Carlgo11** [comment]
@stephengroat apparently my commit automatically dismissed your review. Was it resolved? Otherwise, would you mind re-posting it?

**Comment by Carlgo11** [comment]
@Victor-S-Andrei I changed the logo to the one that [@uidaho](https://twitter.com/uidaho) uses on Twitter and on Facebook.
Text on small images often make them look worse. I'm surprised they use the image you provided as their favicon on [www.uidaho.edu](https://www.uidaho.edu/).

I also retouched some pixels that got distorted when shrinking the image that much.

**Review by mxxcon — APPROVED** [review]

**Pull request merged**